### DISCUSSION TITLE
pkg broke: how to fix?

### DISCUSSION THREAD

**Discussion by John-L** [termux/termux-app#2805] [General]
![Screenshot_20220528-172030](https://user-images.githubusercontent.com/7512058/170843414-9287a6c2-4e2d-43ce-ba75-6e2f00c81f0a.png)

All I've done recently is apply updates. 

Android 12, latest Termux from Google Play. 

Thanks in advance for any insight you can provide.

**Comment by Maxython** [comment]
> Android 12, latest Termux from Google Play.

Termux is no longer updated on google play.  You must install termux from FDroid ([link](https://f-droid.org/packages/com.termux)).

After installation, update all packages:
```
pkg upd -y
```

**Comment by agnostic-apollo** [comment]
Check https://github.com/termux/termux-packages/issues/10778#issuecomment-1134286983 and also shift away from playstore.

https://github.com/termux/termux-app#installation
